copier/template.py

```diff
@@ -435,6 +435,15 @@ def templates_suffix(self) -> str:
             return DEFAULT_TEMPLATES_SUFFIX
         return result
 
+    @cached_property
+    def preserve_symlinks(self) -> bool:
+        """Get whether to preserve symlinks
+
+        By default: `True`
+
+        See [preserve_symlinks][]"""
+        return bool(self.config_data.get("preserve_symlinks", True))
```

Comment:
I think that we should default to `False`.

There are a lot of templates that use this feature as it was implemented before. At least I have a few myself. I also think it's a good default.

```suggestion
        return bool(self.config_data.get("preserve_symlinks", False))
```

This will affect some of the tests.

Reply:
Changed default to false and adjusted tests. Most tests test the default (False) behavior. All tests testing the True behavior are in `test_symlinks.py`